### ISSUE TITLE
localization and language change

### ISSUE THREAD

**Issue by radzik2024** [openhab/openhab-core#4028]
OH version: 4.1.1 and previous
Platform: Windows

Not all descriptions in OpenHab change to the set language - there is a Polish-English mix.

In the regional settings the language is English, but country/region is set to Poland and time-zone set to Warsaw.

I want to use EN language to configure my hub and OH environment but I see mixed Polish and English descriptions as on the picture.

Everything should be consistently set to English.
![language_ENvsPL](https://github.com/openhab/openhab-core/assets/156101620/514a636e-1189-4f45-8ef1-4a289e2c8567)

**Comment by radzik2024** [comment]
![language_ENvsPL2](https://github.com/openhab/openhab-core/assets/156101620/9acb03fd-fb0a-4ee2-a812-30acfa9cee6f)

**Comment by wborn** [comment]
Did you set Polish as the most preferred language in your browser settings?

**Comment by radzik2024** [comment]
Of course, Windows 11, Edge everything is set to Polish language, time zone (Warsaw) etc.
